plugin/trino-hive/src/main/java/io/trino/plugin/hive/authentication/UserGroupInformationUtils.java

```diff
@@ -17,11 +17,11 @@
 
```

Comment:
`Allow public access to UserGroupInformationUtils` > `Make UserGroupInformationUtils class public`